tests/envoygateway/extension_reconciler_test.go

```diff
@@ -171,13 +171,13 @@ var _ = Describe("wasm controller", func() {
 					wasm.AuthServiceName: {
 						Type:        wasm.AuthServiceType,
 						Endpoint:    kuadrant.KuadrantAuthClusterName,
-						FailureMode: wasm.FailureModeDeny,
+						FailureMode: ptr.To(wasm.AuthServiceFailureMode()),
```

Comment:
I'm sure you've already spotted but the `Timeout` is a pointer whereas the `FailureMode` is not which is why the tests are failing
```suggestion
						FailureMode: wasm.AuthServiceFailureMode(),
```